services-core/src/http_server/routing.rs

```diff
@@ -11,6 +11,7 @@ pub struct DefaultRouter {
 }
 
 impl Handler for DefaultRouter {
+    #[allow(clippy::manual_strip)]
```

Comment:
Hmm, weird. Clippy is giving style warnings on macro expansions from foreign crates... That feels wrong.

Reply:
Wasn't it always like this and we had to change ethcontract macros for the same reason?

Reply:
Thats because we use `ethcontract-generate` to generate source and `include!` it, so it was a bit different. Using the proc macro did not, as far as I remember, have this issue.